pkg/ephemeral/spdz.go

```diff
@@ -198,7 +213,23 @@ func (s *SPDZEngine) Activate(ctx *CtxConfig) ([]byte, error) {
 	return nil, errors.New("no MPC parameters specified")
 }
 
-// Compile compiles a SPDZ application.
+func (s *SPDZEngine) getNumberOfThreads() (int, error) {
+	file, err := os.Open(defaultSchedulePath)
```

Comment:
Does this file need to be removed between game executions?
Probably not, since we won't ever get to this point if the compilation fails, would we?

Reply:
Yes. On the one hand, the execution wouldn't reach this point if compilation fails. On the other hand, an ephemeral execution can also be triggered without passing program code to the execution. Doing so would trigger the execution of the previously compiled program - I think this will later on be the default case, where containers will be shipped with a fixed and pre-compiled program and the service with compile option being disabled (to be implemented). With this second option, execution would fail if we delete the  compiled program and/or it's schedule.

Reply:
Yes should have read: Yes, it is not necessary to remove the file, or rather something like: It should not be removed because...